applications/polkadot-faucet.md

```diff
@@ -0,0 +1,217 @@
+- The rise of Play-to-Earn games: The popularity demonstrates the potential for gamified platforms to attract users and incentivize participation.
+
+Similar Projects:
+
+There are no exact replicas of this project within the Polkadot ecosystem. However, some projects share similar goals.
+
+This project focuses on:
+
+- Direct integration with Polkadot: Players experience the functionalities of the Polkadot ecosystem through in-game transactions.
+- Pure Browser-based Game: No additional software downloads are needed, making it more accessible to a wider audience.
+- Focus on Polkadot Education: The game mechanics and rewards are designed to introduce users to Polkadot concepts.
+
+Similar Projects in Other Ecosystems:
+
+Blockchain ecosystems like Ethereum have seen a rise in Play-to-Earn games. However, these projects often have their own token economies and don't directly integrate with Polkadot's parachain infrastructure.
+
+This project aims to bridge the gap by offering a fun and educational gaming experience specifically within the Polkadot ecosystem and fostering user adoption.
+
+## Team
+
+### Team Members
+
+- Olivier Oswald (team leader)
+- pcaversaccio, Stephan Klaus, Stefan Spieler, Rémy Böhler, Alessandro Pittori, more tba (team members)
+
+### Contact
+
+- **Contact Name:** Stephan Klaus
+- **Contact Email:** stephan@blokk.studio
+- **Website:** https://blokk.studio/
+
+### Legal Structure
+
+- **Registered Address:** Landoltstrasse 63, 3007 Bern, Switzerland
+- **Registered Legal Entity:** Apps with love AG
+
+### Team's Experience
+
+Our team boasts a robust blend of blockchain and web development expertise. Rémy, our lead developer, brings years of experience in building websites, Web3 dApps, and browser-based games with a strong emphasis on user engagement. In addition to his extensive web development skills, Rémy also created a decentralized marketplace that earned the Best of Swiss Web Award in 2021. pcaversaccio, our lead blockchain engineer, has extensive experience in developing on the Ethereum network. He is also a long-standing white-hat hacker, contributing to the security and integrity of various blockchain projects.
+
+### Team Code GitHub Repositories
+
+- https://github.com/pcaversaccio
+- https://github.com/ooswald
+- https://github.com/stephan3000
+
+- https://github.com/blokk-studio/assembly-smart-contract
+- https://github.com/blokk-studio/hardhat-project-template-ts
+
+### Team LinkedIn Profiles (If Available)
+
+- https://www.linkedin.com/company/blokkstudio
+- https://www.linkedin.com/in/oswald/
+- https://www.linkedin.com/in/r%C3%A9my-b%C3%B6hler-3565921a3/
+- https://www.linkedin.com/in/stephan-klaus-a3a90b50/
+
+## Development Roadmap
+
+### Overview
+
+- **Total Estimated Duration:** 2 months
+- **Full-Time Equivalent (FTE):** 1,5 FTE
+- **Total Costs:** 12,000 USD
```

Comment:
Please have a look at our updated application template. We offer a flexible (50%+) DOT payment, please provide your requested DOT portion (see [here](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md?plain=1#L127)).